src/_posts/platform/getting-started/2000-01-01-setup-ssh-linux.md

```diff
@@ -14,6 +14,9 @@ ls ~/.ssh
 If the files `id_ed25519` and `id_ed25519.pub` are in the `~/.ssh` folder, you don't
 need to follow this guide, you already have your SSH key.
 
+However, if you have an SSH key but encounter authentication issues, you may need to add it to your SSH agent.
+See the section [Add an existing key to your SSH agent](https://doc.scalingo.com/platform/getting-started/setup-ssh-linux#add-an-existing-key-to-your-ssh-agent) below.
```

Comment:
Links should use `post_url` like explained here: https://github.com/Scalingo/documentation?tab=readme-ov-file#dos